zulipterminal/model.py

```diff
@@ -111,6 +111,29 @@ def get_message_ids_in_current_narrow(self) -> Set[int]:
             current_ids = self.index['search']
         return current_ids.copy()
 
+    def react_to_message(self,
```

Comment:
We should make this method `async` so that the user can still interact with the ui when request is being sent.

Reply:
Done.